web/app/components/since.js

```diff
@@ -1,26 +1,29 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect } from "react";
 
-export default function Since( { props }) {
+export default function Since({ props }) {
   const [time, setTime] = useState(new Date());
-  const since = new Date(props);
+
 
   useEffect(() => {
     const timer = setInterval(() => {
       setTime(new Date());
-    }, 1000)
+    }, 1000);
     return () => {
-      clearInterval(timer)
-    }
-  }, [])
-  
+      clearInterval(timer);
+    };
+  }, []);
+
   // const daysSince = Math.floor((time - since) / (1000 * 24 * 60 * 60));
-  const secondsSince = Math.floor((time - since) / (1000));
-  
-  return ( 
+  const secondsSince = props
+    ? Math.floor((time - props) / 1000)
+    : "Never Eaten";
+
+  return (
+
     <div>
       {/* <p>{since ? `Days since last eaten: ${daysSince < 0 ? 0 : daysSince}` : 'Never eaten' }</p> */}
       {/* <p>{since ? `Seconds since last eaten: ${secondsSince < 0 ? 0 : secondsSince}` : 'Never eaten' }</p> */}
```

Comment:
lets remove these comments. I know we'll soon change this to showing days but we can just adapt what we have without these